### ISSUE TITLE
Minimize, Maximize and Close controls are too small in Windows

### ISSUE THREAD

**Issue by gdsa32** [enzo1982/freac#11]
This is just an usability issue, affecting only the Windows version of fre:ac (I mean, I didn't test it on other platforms, so I'm just supposing...)

The fact is fre:ac's Minimize, Maximize and Close controls are really tiny in Windows. In the attached image, we can compare a native Windows 10 app (Calculator) with fre:ac. 

I think bigger and more spaced icons would greatly improve usability.

![image](https://user-images.githubusercontent.com/39100541/39778455-becb3f68-52dc-11e8-90f3-4fc11d201aac.png)

**Comment by enzo1982** [comment]
Yes, this affects only Windows. On other systems, fre:ac uses the platforms' native titlebars.

I plan to switch to using Windows' native titlebars at some point, too, but only after the upcoming fre:ac 1.1 release.
